app/helpers/metadata_helper.rb

```diff
@@ -1,10 +1,23 @@
 module MetadataHelper
+  include Webpacker::Helper
+
   def meta_tag(key:, value:, opengraph: false)
     return if value.blank?
 
     tag.meta(name: prepend_opengraph(key, opengraph), content: value)
   end
 
+  def image_meta_tags(base_url:, value:, opengraph: true)
+    return if value.blank?
+
+    image_url = "#{base_url}#{asset_pack_path(value)}"
+
+    safe_join([
+      meta_tag(key: "image", value: image_url, opengraph: opengraph),
+      meta_tag(key: "image:alt", value: "Photograph of teaching taking place in a classroom", opengraph: opengraph),
```

Comment:
does the `image:alt` value needs to be dynamic too?

Reply:
Yeah that would be sensible 🤦

Reply:
I've fixed up the change ^